stacks/play.prx.org.yml

```diff
@@ -212,6 +212,40 @@ Resources:
             DNSName: !GetAtt PlayCloudFrontDistribution.DomainName
             HostedZoneId: Z2FDTNDATAQYW2
   # CloudFront
+  PlayCloudFrontCachePolicy:
+    Type: AWS::CloudFront::CachePolicy
+    Properties:
+      CachePolicyConfig:
+        Comment: !Sub Policy for ${AWS::StackName}
+        DefaultTTL: 86400
+        MaxTTL: 31536000
+        MinTTL: 0
+        Name: !Sub ${AWS::StackName}-CachePolicy
+        ParametersInCacheKeyAndForwardedToOrigin:
+          CookiesConfig:
+            CookieBehavior: none
+          EnableAcceptEncodingBrotli: true
+          EnableAcceptEncodingGzip: true
+          HeadersConfig:
+            HeaderBehavior: none
+          QueryStringsConfig:
+            QueryStringBehavior: whitelist
+            QueryStrings:
+              - url
+  PlayCloudFrontOriginRequestPolicy:
+    Type: AWS::CloudFront::OriginRequestPolicy
+    Properties:
+      OriginRequestPolicyConfig:
+        Comment: !Sub Policy for ${AWS::StackName}
+        CookiesConfig:
+          CookieBehavior: none
+        HeadersConfig:
+          HeaderBehavior: none
+        Name: !Sub ${AWS::StackName}-OriginRequestPolicy
+        QueryStringsConfig:
+          QueryStringBehavior: whitelist
+          QueryStrings:
+            - url
```

Comment:
Just thinking this through (even though it's the same setting we had before) ...

This will cache any RSS proxied by the express server.  But other pages with query params (the embed player angular app itself) ... it will just cache 1 copy of that HTML.  But that's fine, since all the HTML is the same, and js/angular is parsing the query params itself.